cmd/release/utils/values/searchprs.go

```diff
@@ -12,34 +12,53 @@ const (
 	baseQuery = "repo:aws/eks-distro is:pr is:merged"
 )
 
-func GetChangelogPRs(releaseVersion string) (string, error) {
+func GetChangelogPRs(releaseVersion string, overrideNumber int) (string, error) {
 	githubClient := github.NewClient(nil)
 
 	ctx := context.Background()
-	opts := &github.SearchOptions{}
+	opts := &github.SearchOptions{Sort: "updated"}
 	//Get the date of the last document release for the release version
-	prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged label:release label:documentation repo:aws/eks-distro label:" + releaseVersion, opts)
+	prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged label:release label:documentation repo:aws/eks-distro label:"+releaseVersion, opts)
```

Comment:
I still don't get how this works with `label:documentation` but it does. Is it an OR search?

Reply:
oh wait. Never mind. It's dealt with below